tests/solvers/flow_control_01.cc

```diff
@@ -34,42 +34,40 @@ void
 test()
 {
   Parameters::DynamicFlowControl flow_control_parameters;
-  flow_control_parameters.flow_direction = 0;
-  flow_control_parameters.flow_rate_0    = -10;
-  flow_control_parameters.beta_0         = 20;
+  flow_control_parameters.flow_direction     = 0;
+  flow_control_parameters.average_velocity_0 = 10;
+  flow_control_parameters.beta_0             = 20;
+  flow_control_parameters.alpha              = 0.5;
 
-  std::pair<double, double> flow_rate(0.0, 3.1416);
-  double                    dt = 0.1;
+  double average_velocity = 0.0;
+  double dt               = 0.1;
 
   FlowControl<3> flow_control(flow_control_parameters);
 
   Tensor<1, 3> beta;
 
   deallog << "*************************************************" << std::endl;
-  deallog << " Time step :         " << dt << std::endl;
-  deallog << " Initial beta :      " << flow_control_parameters.beta_0
-          << std::endl;
-  deallog << " Desired flow rate : " << flow_control_parameters.flow_rate_0
+  deallog << " Time step:         " << dt << std::endl;
+  deallog << " Initial beta:      " << flow_control_parameters.beta_0
           << std::endl;
+  deallog << " Desired average velocity: "
```

Comment:
Desired or target?

Reply:
It means the same to me

Reply:
It means the same, but I just want the vocabulary to be consistent throughout the code.